en/assembly/quick_start_cuav_v5_nano.md

```diff
@@ -72,16 +75,21 @@ The figure below shows how you can access your remote receiver (please find the
 ## Spektrum Satellite Receivers
 
 The V5 nano has a dedicated DSM cable.
-If using a Spektrum satellite receiver, this should be connected to the flight controller DSM/SBUS/RSSI interface. 
+If using a Spektrum satellite receiver, this should be connected to the flight controller `DSM/SBUS/RSSI` interface.
 
 
 ## Power
 
-The v5 nano kit includes HV\_PM, which supports 2~10S LiPo battery.
-Please connect the 6pin connector of the HW\_PM module to the flight control `Power` interface.
+The *v5 nano* kit includes the *HV\_PM* module, which supports 2~10S LiPo batteries.
+Connect the 6pin connector of the *HW\_PM* module to the flight control `Power` interface.
+
+> **Warning** The supplied power module is unfused (see [this issue](../flight_controller/cuav_v5_nano.md#issue_pm_unfused)).
```

Comment:
@davids5 Minimal note about unfused module here, links to associated issue in main doc.

Reply:
@hamishwillee there is more to it then that. On a FMUV2, FMUV5 we have over current protection on the 5 Volt Peripheral and 5 Volt high power. This limits the current to 1 A and 1.5 A respectively. On FMUv4 we have fuses. The NANO has neither. Therefore 4 A can be delivered to the connectors with  V5+ listed as pin 1 (that are rated at 1 A).  This can pose a potential danger. Please make it Clear 1) that power needs to be off while connecting peripherals . 2) Improper wiring can lead to equipment or personal damage.